sdk/communication/Shared/src/CommunicationIdentifierModel.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.Communication
```

Comment:
We modify swagger locally to generate these classes.  These generated class will be in chat swagger, right? And you copy the generated code and include in this PR. My question is "Shouldn't these classes in the 'Chat' directory? instead of 'Shared' directory?

Reply:
Once chat starts using this, we will remove this file altogether. For now, I added it to make sure we can test things